tests/run.bats

```diff
@@ -19,6 +19,7 @@ load helpers
 	buildah run        $cid cp /tmp/randomfile /tmp/other-randomfile
 	test -s $root/tmp/other-randomfile
 	cmp ${TESTDIR}/randomfile $root/tmp/other-randomfile
+	buildah run $cid -- rm -f /tmp/randomfile /tmp/other-randomfile
```

Comment:
So you want
run buildah run $cid --  echo -n foobar
[  "$output" == "foobar" ]
Do you also want a negative test
run buildah run $cid  echo -n foobar
[  $status != 1 ]